holidays/countries/pitcairn_islands.py

```
@@ -0,0 +1,69 @@
+    def _add_holiday_jan_23(self, name: str):
+        """Add Bounty Day (January 23)."""
+        self._add_holiday(name, JAN, 23)
+
+    def _add_kings_birthday(self, name: str):
+        """Add King's Birthday (2nd Saturday in June)."""
+        self._add_nth_weekday_of_month(name, 2, 5, JUN)  # 2nd Saturday in June (5 = Saturday)
+
+
+class PN(PitcairnIslands):
+    """Alias for PitcairnIslands."""
+    pass
```

Ruff (0.11.9)

No newline at end of file

Add trailing newline

(W292)